CONTRIBUTING.md

```diff
@@ -384,39 +396,58 @@ Here is a skeleton predicate script:
 # * we access any undefined variable.
 set -eu
 
-# Print out Rust backtraces on panic. Useful for minimizing a particular panic.
-export RUST_BACKTRACE=1
+# Invoke the general purpose predicate script that comes in the
+# `bindgen` repository.
+path/to/rust-bindgen/csmith-fuzzing/predicate.py \
+    $PREDICATE_FLAGS \
```

Comment:
There are no `$PREDICATE_FLAGS` and I think it would be nice if we would provide default one, or at least placeholder.